### PR TITLE
More troubleshot docs

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: content/docs/troubleshoot/analyzers/config-management.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+categories: [ "Replicated Troubleshoot" ]
+index: ["docs/troubleshoot", "docs"]
+aliases: [ /docs/troubleshoot/analyzers ]
+icon: "replicatedTroubleshoot"
+gradient: "orangeToOrange"
+---
+
+# Configuration Management Software
+
+Popular configuration management tools such as Chef, Puppet and Ansible are used to help mamangement large numbers of servers using automation.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/config-management.md]: `manage`

File: content/docs/troubleshoot/analyzers/config-management.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+When setting up a server, especially a black box, appliance, it's important to have the server running with known-compatible software and dependencies. When an appliance is provisioned in an environment, occaisionally these configuration management tools will run and attempt to bring all of the software to known versions to match other servers in the environment. This can sometimes cause changes made locally to be reverted later.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/config-management.md]: Something complicated is going ton with comas in the beginning of this sentence.

File: content/docs/troubleshoot/analyzers/disk-space.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+categories: [ "Replicated Troubleshoot" ]
+index: ["docs/troubleshoot", "docs"]
+aliases: [ /docs/troubleshoot/analyzers ]
+icon: "replicatedTroubleshoot"
+gradient: "orangeToOrange"
+---
+
+# Disk Pressure
+
+One of the most common causes of service outages on appliances is running out of disk space. If a server is low on disk space, the analyzers will show a warning message and make this visible and easy to detect.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/disk-space.md]: Is this just space or inodes too?

File: content/docs/troubleshoot/analyzers/docker-icc.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+aliases: [ /docs/troubleshoot/analyzers ]
+icon: "replicatedTroubleshoot"
+gradient: "orangeToOrange"
+---
+
+# Docker ICC
+
+Some installations of Docker will disable inter-container communication, preventing containers on the server from communicating with each other over the host networking stack. This is a default setting for enterprise distrubutions such as RedHat Enterprise Linux.
+
+When debugging problems, this might be be obvious, and if ICC is disabled, a message will be displayed in the support bundle analyzers to help make the engineer aware of the fact that inter-container communication will be blocked.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/docker-icc.md]: `be be` -> `not be`

File: content/docs/troubleshoot/analyzers/overview.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+index: ["docs/troubleshoot", "docs"]
+aliases: [ /docs/troubleshoot/analyzers ]
+icon: "replicatedTroubleshoot"
+gradient: "orangeToOrange"
+hideFromList: true
+---
+
+# Analyzers
+
+After uploading a support bundle to the [Vendor Portal](https://vendor.replicated.com/troubleshoot), a series are analyzers will run to extractr insights from the logs and other files included.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/overview.md]: `a series are analyzers`
`extractr `

File: content/docs/troubleshoot/analyzers/overview.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+![Support Bundle Analyzers](/images/troubleshoot/analyzers.png)
+
+Many different analyzers are run, and attempt to find known incompatibilities, misconfigured services and environment issues that might4 be causing problems. Additionally, commonly accessed information such as version numbers, server load, and more are displayed here to help reduce the need to dig through log files to find this information.
```
Comment (divolgin) [inline content/docs/troubleshoot/analyzers/overview.md]: `might4`

File: content/docs/troubleshoot/analyzers/config-management.md (hunk 2 above)
Comment (marccampbell) [reply content/docs/troubleshoot/analyzers/config-management.md]: hmm, yeah, you, might be, right.  this feels, off.

File: content/docs/troubleshoot/analyzers/disk-space.md (hunk 3 above)
Comment (marccampbell) [reply content/docs/troubleshoot/analyzers/disk-space.md]: just space right now, rihgt?

**Pull request merged**